constants.py

```diff
@@ -1,24 +1,54 @@
+import json
+
 import variables as var
 
+default_lang_dict = {}
+lang_dict = {}
+
+
+def load_lang(lang):
+    global lang_dict, default_lang_dict
+    with open("lang/en_US", "r") as f:
+        default_lang_dict = json.load(f)
+    with open(f"lang/{lang}", "r") as f:
+        lang_dict = json.load(f)
+
+
+def tr_cli(option, *argv, **kwargs):
+    try:
+        if option in lang_dict['cli'] and lang_dict['cli'][option]:
+            string = lang_dict['cli'][option]
+        else:
+            string = default_lang_dict['cli'][option]
+    except KeyError:
+        raise KeyError("Missed strings in language file: '{string}'. ".format(string=option))
+    return _tr(string, *argv, **kwargs)
+
 
-def strings(option, *argv, **kwargs):
+def tr_web(option, *argv, **kwargs):
     try:
-        string = var.config.get("strings", option)
```

Comment:
You should keep it for "user can override a string" purpose :
- New PR can just add one string into the config until a translator guy add a new term into traduora
- People can still change some strings for fun.